test/unit/pages/api/plans/index.test.js

```diff
@@ -62,7 +67,10 @@ describe('Create Plan Api', () => {
       error: 'could not create plan with firstName=James, lastName=Bond'
     };
 
-    const createPlan = jest.fn(x => {
+    const createPlan = {
+      execute: jest.fn()
+    };
+    createPlan.execute.mockImplementation(() => {
```

Comment:
This should be the same as
```
execute: jest.fn(() =>  { throw new Error('bang'); });
```

Reply:
that's what we had in there initially but it was giving us weird babel errors (on bogdan's machine anyway). i do wonder what's going on there, esp after the Boolean() fiasco